pyrit/datasets/ccp_sensitive_prompts_dataset.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import Path
+from typing import Literal, Optional
+
+from .dataset_helper import FILE_TYPE_HANDLERS, fetch_examples
+from ..models import SeedPromptDataset
```

Comment:
```suggestion
from pyrit.models import SeedPromptDataset
```
Same with other imports. That makes it explicit rather than than relative